### ISSUE TITLE
Firebase HTTP Cloud function query data undefined

### ISSUE THREAD

**Issue by abdullahamin0111** [firebase/firebase-tools#1345]
I've been trying to run the firebase cloud HTTP function locally on my computer.
When I try to send the query data to the http function locally then it is undefined.

**Windows 10 is being used.**

The function is:
```typescript
export const checkHttp = functions.https.onRequest((req,res)=>{
    const id = req.query.id == undefined ? "undefined" : req.query.id
    res.send({
        "ID":id
    })
})
```

The command I used to start the emulation for functions is:
**firebase serve --only functions**

The url for local emulation is returned as
http://localhost:5000/..../checkHttp

Now even if I pass the url with the following 
http://localhost:5000/..../checkHttp?id=123 //anything

The response is:
```JSON
{ "ID" : "undefined" }
```
But when I deploy that function and use that deployed url then it works fine

for example if I run the same query on the deployed function then the result is correct:
URL:     https://us-central1-..../checkHttp?id=123
Response:    
```JSON
{ "ID" : 123 }
```

The package.json is as follows 
```
{
  "name": "functions",
  "scripts": {
    "lint": "tslint --project tsconfig.json",
    "build": "tsc",
    "serve": "npm run build && firebase serve --only functions",
    "shell": "npm run build && firebase functions:shell",
    "start": "npm run shell",
    "deploy": "firebase deploy --only functions",
    "logs": "firebase functions:log"
  },
  "engines": {
    "node": "10"
  },
  "main": "lib/index.js",
  "dependencies": {
    "firebase-admin": "8.0.0",
    "firebase-functions": "^2.3.1",
    "grpc": "^1.21.1"
  },
  "devDependencies": {
    "firebase-functions-test": "^0.1.6",
    "tslint": "^5.12.0",
    "typescript": "^3.2.2"
  },
  "private": true
}
```

**Comment by thechenky** [comment]
@abeisgoat @samtstern would you mind taking a look at this? Thanks!

**Comment by abdullahamin0111** [comment]
The response works perfectly when firebase-tools are downgraded....

Version that was giving error was: **6.10.0**
The version that now runs perfectly: **6.8.0**

Thanks.

**Comment by samtstern** [comment]
This was fixed by #1332 and will be included in the next release.  Thanks for your patience!
